mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/persistence/auth/firebase/FirebaseAuth.kt

```diff
@@ -38,28 +39,41 @@ class FirebaseAuth(private val actual: ActualFirebaseAuth) : Auth {
    */
   private suspend fun authenticate(
       block: suspend () -> AuthResult?,
+      mode: AuthenticationMode,
```

Comment:
Keep the lambda parameter last so we can use some trailing lambda notation (like `authenticate(SignIn) { /* Something */ }`).

```suggestion
      mode: AuthenticationMode,
      block: suspend () -> AuthResult?,
```

Reply:
Fixed in 12d715c